iam-ecs.tf

```diff
@@ -74,3 +74,72 @@ EOF
 
   tags = "${var.apiary_tags}"
 }
+
+resource "aws_iam_role_policy" "ecr_permission_for_task_exec" {
+  count = "${var.instance_type == "ecs" ? 1 : 0}"
+  name  = "ecr-permission"
+  role  = "${aws_iam_role.apiary_task_exec.id}"
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ecr:BatchGetImage",
+                "ecr:GetDownloadUrlForLayer",
+                "ecr:GetAuthorizationToken"
+            ],
+            "Resource": "*"
```

Comment:
Should the policy be set on all resources? Or should instead be configurable and that way you can choose to do it on all resources or a subset?